src/lib/utils/balancer/wrapper.ts

```diff
@@ -47,7 +49,7 @@ export const getWrapOutput = (
   if (wrapper === wstETH) {
     return wrapType === WrapType.Wrap
       ? getWstETHByStETH(wrapAmount)
```

Comment:
Doesn't this function need to be updated in the same way too?

Reply:
It would have to output the inverse of `getRate()`

Reply:
fixed

Reply:
I meant don't we have to refactor the original implementation of `getWstETHByStETH` so that the amount is converted base on rateProvider.getRate() instead of based on a call to the wstETH contract?